centro-htb-validator.js

```diff
@@ -41,13 +41,32 @@ var partnerValidator = function (configs) {
                     '*': {
                         type: 'object',
                         properties: {
-                            placementId: {
+                            placement: {
                                 type: 'string',
                                 minLength: 1
+                            },
+                            size: {
+                                type: 'array',
+                                exactLength: 2,
+                                items: { type: 'number' }
+                            },
+                            page_url: {
+                                type: 'string',
+                                optional: true
+                            },
+                            timeout: {
```

Comment:
timeout should not be necessary and be removed , is there any special reason that this is being added?

Reply:
removed confirmed